monai/transforms/compose.py

```
@@ -262,6 +262,48 @@ def __init__(
+            # Check common attribute patterns for wrapped transforms
+            for attr_name in dir(obj):
+                # Skip magic methods and common non-transform attributes
+                if attr_name.startswith("__") or attr_name in ("transforms", "transform"):
+                    continue
+                try:
+                    attr = getattr(obj, attr_name, None)
+                    if attr is not None and isinstance(attr, TraceableTransform) and not isinstance(attr, Compose):
+                        # Recursively set group on nested transforms
+                        set_group_recursive(attr, gid)
+                except Exception:
+                    pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)